src/helper-macros.js

```diff
@@ -1,12 +1,6 @@
 let if = macro {
     rule { $cond:expr then $then:expr else $else:expr } => {
-        (function() {
-            if ($cond) {
-                return $then
-            } else {
-                return $else
-            }
-        }.bind(this))()
+        (($cond) ? ($then) : ($else))
```

Comment:
just fyi the defensive `()` around each pattern aren't necessary. unlike c macros, sweet.js does the right thing and puts them there for you.